sgp4/tests.py

```diff
@@ -207,6 +207,19 @@ def test_tle_export():
         if satrec.satnum not in expected_errs_line2:
             assertEqual(out_line2, line2)
 
+def test_export_tle_raises_error_for_out_of_range_angles():
+    # See https://github.com/brandon-rhodes/python-sgp4/issues/70
+    sat = Satrec()
+
+    for angle in 'inclo', 'nodeo', 'argpo', 'mo':
+        WRONG_VANGUARD_ATTRS = VANGUARD_ATTRS.copy()
```

Comment:
For a local variable, try flipping to lowercase — so: `wrong_vanguard_attrs` for the value here.